deployment/CE07SHSM_Deploy.csv

```diff
@@ -28,23 +28,23 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
+1,CE07SHSM-SBD11-04-VELPTA000,CGINS-VELPTA-11401,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,1.3,87,
+1,CE07SHSM-SBD11-06-METBKA000,CGINS-METLGR-00008,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,0,87,nominal_depth
 1,CE07SHSM-SBD12-00-DCLENG000,OL000412,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,0,87,nominal_depth
 1,CE07SHSM-SBD12-03-HYDGN0000,OL000407,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,0,87,nominal_depth
-1,CE07SHSM-SBD12-04-PCO2AA000,CGINS-PCO2AA-34196,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,,87,
+1,CE07SHSM-SBD12-04-PCO2AA000,CGINS-PCO2AA-34196,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,0.7,87,
 1,CE07SHSM-SBD12-05-WAVSSA000,CGINS-WAVSSA-05271,2015-04-15T17:15:00,2015-09-23T15:08:00,46.98611,-124.56611,CGMCE-07SHSM-00001,OC1503D,,1,,,,,0,87,nominal_depth
-2,CE07SHSM-MFC31-00-CPMENG000,OL000422,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,500,90,nominal_depth
-2,CE07SHSM-MFD35-00-DCLENG000,OL000427,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,500,90,nominal_depth
-2,CE07SHSM-MFD35-01-VEL3DD000,CGINS-VEL3DD-12266,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD35-02-PRESFB000,CGINS-PRESFB-01396,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD35-04-ADCPTC000,CGINS-ADCPTC-22114,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD35-05-PCO2WB000,CGINS-PCO2WB-C0119,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD35-06-PHSEND000,CGINS-PHSEND-P0123,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD37-00-DCLENG000,OL000428,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,500,90,nominal_depth
-2,CE07SHSM-MFD37-01-OPTAAD000,CGINS-OPTAAD-00182,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD37-03-CTDBPC000,CGINS-CTDBPC-50006,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,87,90,nominal_depth
-2,CE07SHSM-MFD37-03-DOSTAD000,CGINS-DOSTAD-00479,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,,90,
+2,CE07SHSM-MFC31-00-CPMENG000,OL000422,2015-10-13T17:02:00,2016-05-13T22:15:00,46.98749,-124.56871,CGMCE-07SHSM-00002,OC1510A,,1,,,,,90,90,nominal_depth
```

Comment:
These depths should be 87 meters, not 90.

Reply:
Done.